middleware/lib/claim-client.js

```diff
@@ -11,9 +11,10 @@ const {
   Secp256k1Context: Context,
 } = require('sawtooth-sdk/signing/secp256k1');
 
+const createFamily = require('../../utils/family');
+const { loadType } = require('../../utils/proto');
+
```

Comment:
Lol wow...

Reply:
Wow what?

Also, either approve this or don't, ya jabroni.